src/battle_interface.c

```diff
@@ -2340,11 +2340,11 @@ static s32 CalcNewBarValue(s32 maxValue, s32 oldValue, s32 receivedValue, s32 *c
 
     if (maxValue < scale) // handle cases of max var having less pixels than the whole bar
     {
-        s32 toAdd = Q_24_8(maxValue) / scale;
```

Comment:
I think keeping the variable shadowing might be fine here.  Changing it to a Frankenstein name isn't worth it, in my opinion.  `delta` might be a better name than `toAdd` for the whole function..